apisix/http/service.lua

```diff
@@ -24,6 +26,48 @@ local _M = {
 }
 
 
+local function filter(service)
```

Comment:
https://github.com/apache/incubator-apisix/pull/1402

we can merge this PR first

Reply:
@qiujiayu you can rebase your branch with master, #1402 has been merged.

Reply:
done